pokemongo_bot/cell_workers/move_to_map_pokemon.py

```diff
@@ -245,7 +245,7 @@ def snipe(self, pokemon):
                     exists = True
 
                     # Also, if the IDs arent valid, update them!
-                    if not pokemon['encounter_id'] or not pokemon['spawnpoint_id']:
+                    if not pokemon['encounter_id'] or not pokemon['spawn_point_id']:
```

Comment:
Nah, not that simple. Fixes the "does not exist" error, but still fails to actually catch anything.